### PR TITLE
cockpit-components-terminal: change onFocusIn/OnFocusOut to onFocus/onBlur

### PR DESCRIPTION
**Pull request by mareklibra**
Part of #9263 PR-split, see individual commits for more info.

### CODE REVIEW

File: pkg/lib/cockpit-components-terminal.jsx [hunk 1]
```diff
@@ -110,8 +110,8 @@
             return <div ref="terminal"
                         key={this.state.terminal}
                         className="console-ct"
-                        onFocusIn={this.onFocusIn}
-                        onFocusOut={this.onFocusOut} />;
+                        onFocus={this.onFocusIn}
+                        onBlur={this.onFocusOut} />;
```
Comment (martinpitt) [inline pkg/lib/cockpit-components-terminal.jsx]: These events don't work with react-lite, the handler is never called. So either both properties should be set for a transitional period, or this is blocked on actually switching to React. At that point the handler methods should be renamed accordingly, too.

**Comment by mareklibra** [comment]
Let's keep it blocked till #9263 lands.

**Comment by martinpitt** [comment]
#9263 landed now, unblocking.

**Comment by martinpitt** [comment]
Rebased to current master and adjusted commit log a bit. This might even be a regression from the react switch, thus priority.

**Pull request merged**